js/event.ts

```diff
@@ -0,0 +1,285 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+import * as domTypes from "./dom_types";
+
+// WeakMaps are recommended for private attributes (see MDN link below)
+// tslint:disable-next-line:max-line-length
+// https://developer.mozilla.org/en-US/docs/Archive/Add-ons/Add-on_SDK/Guides/Contributor_s_Guide/Private_Properties#Using_WeakMaps
+export const eventAttributes = new WeakMap();
+
+export class EventInit implements domTypes.EventInit {
+  bubbles = false;
+  cancelable = false;
+  composed = false;
+
+  constructor({bubbles=false, cancelable=false, composed=false} = {}) {
```

Comment:
Not sure this has gone through prettify...

Reply:
@acconrad run `./tools/format.py`

Reply:
Somehow this isn't working for me, I'm getting a `file not found` error but I def have recursive modules set up for the `build/` folder... 🤔

Reply:
all set